tests/integration/tools/mongodb/metadata/connect.test.ts

```diff
@@ -26,32 +26,32 @@ describe("Connect tool", () => {
         await cluster.close();
     });
 
-    describe("with default config", () => {
-        it("should have correct metadata", async () => {
-            const tools = await client.listTools();
-            const connectTool = tools.tools.find((tool) => tool.name === "connect");
-            expect(connectTool).toBeDefined();
-            expect(connectTool!.description).toBe("Connect to a MongoDB instance");
-            expect(connectTool!.inputSchema.type).toBe("object");
-            expect(connectTool!.inputSchema.properties).toBeDefined();
-
-            const propertyNames = Object.keys(connectTool!.inputSchema.properties!);
-            expect(propertyNames).toHaveLength(1);
-            expect(propertyNames[0]).toBe("connectionStringOrClusterName");
-
-            const connectionStringOrClusterNameProp = connectTool!.inputSchema.properties![propertyNames[0]] as {
-                type: string;
-                description: string;
-            };
-            expect(connectionStringOrClusterNameProp.type).toBe("string");
-            expect(connectionStringOrClusterNameProp.description).toContain("MongoDB connection string");
-            expect(connectionStringOrClusterNameProp.description).toContain("cluster name");
-        });
+    it("should have correct metadata", async () => {
+        const tools = await client.listTools();
+        const connectTool = tools.tools.find((tool) => tool.name === "connect");
```

Comment:
```suggestion
        const { tools } = await client.listTools();
        const connectTool = tools.find((tool) => tool.name === "connect")!;
```
very nit (also with assertion here we'd be fine below)